cpp/CMakeLists.txt

```diff
@@ -787,7 +831,7 @@ target_compile_definitions(cudf PUBLIC SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_${LOGGIN
 # - link libraries --------------------------------------------------------------------------------
 
 # link targets for cuDF
-target_link_libraries(cudf arrow arrow_cuda nvrtc ${CUDART_LIBRARY} cuda ${ZLIB_LIBRARIES} ${Boost_LIBRARIES})
+target_link_libraries(cudf arrow arrow_cuda nvrtc ${CUDART_LIBRARY} ZLIB::ZLIB Boost::filesystem)
```

Comment:
Since we're now using the `ZLIB::ZLIB` and `Boost::filesystem` targets can we remove the usage of `${ZLIB_INCLUDE_DIR}`, `${Boost_INCLUDE_DIR}`, `${ZLIB_LIBRARY}`, and `${Boost_LIBRARY}`?